src/index.js

```diff
@@ -0,0 +1,41 @@
+import './style.css';
+
+const taskList = [
+  {
+    description: 'task1',
+    completed: 0,
+    index: 0,
+  },
+  {
+    description: 'task2',
+    completed: 0,
+    index: 0,
+  },
+  {
+    description: 'task3',
+    completed: 0,
+    index: 0,
+  },
+];
```

Comment:
- [ ] Kindly make sure that the `completed` property is of type `boolean` True/False.
- [ ] Each task should have a unique index, kindly update them (indexes will be used in next milestone).